boilerplate/openshift/golang_osd_hive_operator/codecov.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Same comment on DRY'ing this up into a common convention for operators.

Reply:
See above.